From 3cf99fb519595e3934882ac9ea32409badc6b57a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Arkadiusz=20Mi=C5=9Bkiewicz?= <arekm@maven.pl>
Date: Fri, 27 Jun 2014 13:50:25 +0200
Subject: [PATCH] Enable SCSI_CONSTANTS; The error messages regarding your SCSI
 hardware will be easier to understand if you say Y here.

---
 kernel-multiarch.config | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel-multiarch.config b/kernel-multiarch.config
index ea15c17..8aee3c6 100644
--- a/kernel-multiarch.config
+++ b/kernel-multiarch.config
@@ -4513,7 +4513,7 @@ CHR_DEV_SG all=m
 CHR_DEV_SCH all=m
 SCSI_ENCLOSURE all=m
 SCSI_MULTI_LUN all=y
-SCSI_CONSTANTS all=n
+SCSI_CONSTANTS all=y
 SCSI_LOGGING all=y
 SCSI_SCAN_ASYNC all=y
 SCSI_WAIT_SCAN all=m
-- 
2.0.0

